FAQ.md

```diff
@@ -20,9 +55,13 @@ Such as:
 
 Please see [here](https://github.com/lucko/LuckPerms/wiki/Storage-system-errors).
 
-### I use SpongeForge and when I install LuckPerms do plugins no longer work for OPs
-This is an intentional behaviour of SpongeForge. As soon as a permission plugin, in this case LuckPerms, is installed will it disable the OP system.  
-To fix this give the groups and users the required permissions for the plugins.
+***
+
+### I use SpongeForge and when I install LuckPerms I lose all my permissions, why?
+This is an intentional behaviour of SpongeForge. As soon as a permission plugin is installed it will disable the OP system.  
```

Comment:
"As soon as a permission plugin is installed it will disable the OP system."?
Wouldn't "As soon as a permission plugin is installed will it disable the OP system." or at least need a comma after the "installed"? Not an expert in grammar here.

Reply:
"As soon as a permission plugin is installed will it disable the OP system" is a question.
The current one seems fine for me, maybe a comma after "installed".

Reply:
Yeah. A comma seems good because when you read it out loud will you notice how weird it may sound without a small pause in between.